geo/src/main/res/layout/selection_map_layout.xml

```diff
@@ -103,6 +103,7 @@
         android:layout_height="match_parent"
         android:background="?colorSurface"
         app:behavior_hideable="true"
+        android:maxWidth="10000dp"
```

Comment:
Could we do this programmatically and use `Integer.MAX_VALUE` instead? It [looks like](https://stackoverflow.com/questions/28205014/android-remove-max-width-attribute-from-text-view) that's what Android does internally.

I'd also be keen to make it more explicit what's going on here. We already have our own implementation of `BottomSheetBehavior`, so we could add a method like `setMaxWidth(width: Int?)` that just "disables" the max width on the underlying (actual) `BottomSheetBehavior` (by setting the max width to the `MAX_VALUE`).